src/main/kotlin/com/exactpro/th2/rptdataprovider/handlers/SearchEventsHandler.kt

```diff
@@ -288,7 +294,10 @@ class SearchEventsHandler(
             val lastEventId = AtomicLong(0)
             val scanCnt = AtomicLong(0)
 
-            val startTimestamp = getStartTimestamp(request)
+            val resumeFromEvent = request.resumeFromId?.let {
+                eventProducer.fromId(ProviderEventId(request.resumeFromId))
+            }
+            val startTimestamp = resumeFromEvent?.startTimestamp ?: request.startTimestamp!!
```

Comment:
```suggestion
            val startTimestamp = resumeFromEvent?.startTimestamp ?: requireNotNull(request.startTimestamp) {
                "Either 'resumeFromEvent' or 'startTimestamp' must be set in the request"
            }
```

Reply:
This case is checked when we create a request.
https://github.com/th2-net/th2-rpt-data-provider/blob/c5f7af3ab4279f1b5f4f0dbc717a9bab346dffaa/src/main/kotlin/com/exactpro/th2/rptdataprovider/entities/requests/SseEventSearchRequest.kt#L117
We need check in twice?